src/main/java/fr/openmc/core/features/contest/menu/TradeMenu.java

```diff
@@ -132,7 +139,7 @@ public TradeMenu(Player owner) {
                                     player.getInventory().addItem(shell_contestItem);
                                 }
 
-                                ItemStack newshell_contestItem = CustomStack.getInstance("contest:contest_shell").getItemStack();
+                                ItemStack newshell_contestItem = CustomStack.getInstance(namespaceShellContest).getItemStack();
```

Comment:
Ce nom de variable ne respecte pas les règles de contribution.
https://github.com/ServerOpenMC/PluginV2/blob/master/CONTRIBUTING.md#-style-de-code
```suggestion
                                ItemStack newShellContestItem = CustomStack.getInstance(namespaceShellContest).getItemStack();
```

Reply:
Oula mais oui je l'avais fais quand je débutais et ouais j'ai fais n'importe quoi

Reply:
Pas de problème, les revues sont là pour avoir un deuxième regard sur le code et rattraper nos oublis